.travis.yml

```diff
@@ -28,7 +28,7 @@ cache:
     - $HOME/.jest-cache
     - $HOME/.npm
     - $HOME/.nvm/.cache
-    - $HOME/.wp-cli/.cache
+    - $HOME/.wp-cli/.cachec
```

Comment:
@swissspidy Is there a stray `c` here?

Reply:
Whoops, indeed! No idea how that got there. Thanks!